src/index.html

```diff
@@ -11,6 +11,37 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="#"
+      data-qa="big"
+      class="form_base"
+    >
```

Comment:
```suggestion
    <form 
      action="#"
      data-qa="big"
      class="form_base"
    >
```
Consider fixing indents according to the code style guide